src/app/shared/model/UserPolicy.model.ts

```diff
@@ -1,4 +1,5 @@
 export interface UserPolicyModel {
+  id: number;
```

Comment:
add id param to the model because id is required when handle the rest api services